src/Application/Actions/UpdatePaymentMethod.php

```diff
@@ -21,6 +21,7 @@ class UpdatePaymentMethod extends Action
         'Your card has expired.',
         'Your card does not support this type of purchase.',
         'Your card\'s expiration month is invalid.',
+        'Invalid account',
```

Comment:
```suggestion
        'Invalid account.',
```

I think we need Stripe's punctuation as we're after an exact match upon comparison.

Reply:
You're right, thanks.